packages/jest-mock/__typetests__/Mocked.test.ts

```diff
@@ -6,7 +6,7 @@
  */
 
 import {describe, expect, test} from 'tstyche';
-import type {MockInstance, Mocked} from 'jest-mock';
+import type {MockInstance, Mocked} from '../src/index';
```

Comment:
Imports should stay as they were. Types of the build artefact are tested on purpose, to make sure types are correct on the user side.

That is why you have to build the repository (or rebuild it after each change) to be able to run the type tests locally.

Reply:
added this here also #15181